source/iostreams.tex

```diff
@@ -4729,6 +4737,64 @@
 \end{codeblock}
 \end{itemdescr}
 
+\begin{itemdecl}
+basic_istream& operator>>(@\placeholder{extended-floating-point-type}@& val);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
```

Comment:
Why aren't we using any Elements here? Shouldn't this also be Effects:, and there should be a Returns:?

@dkolsen-pgi?

Reply:
Because none of the other `operator>>` descriptions in [iostream.formatted.arithmetic] use any Elements.  I was just following the existing convention in that section.

Reply:
Interesting, thanks. Let me open an issue for that, but this certainly makes sense as is.